contracts/TiramisuSavingsClub.sol

```diff
@@ -47,8 +50,8 @@ contract TiramisuSavingsClub {
     /// @dev Reverts if any of the member addresses currently belong to another group (not allowed)
     /// @return group id
     function createGroup(address[] memory _members, string[] memory _names, uint _ownerIndex) public returns (uint) {
-        require(_members.length > 0, "Can't create empty group");
-        require(_members.length == _names.length, "_members/_names len should match");
+        require(_members.length > 0, "Cannot create an empty group");
```

Comment:
Can we revert these string changes? I shortened them intentionally to pass the solhint message length requirement

Reply:
Same feedback for all of the require() strings